spec.bs

```diff
@@ -1243,6 +1248,7 @@ The following data structures are shared between registration and authentication
         required USVString displayName;
         required USVString icon;
         boolean iconMustBeShown = true;
+        USVString details;
```

Comment:
As an alternate approach, we could instead extend the existing `displayName` concept, into:

```
    dictionary PaymentCredentialInstrument {
        required (USVString or PaymentCredentialInstrumentDetails) displayName;
        required USVString icon;
        boolean iconMustBeShown = true;
   }

  dictionary PaymentCredentialInstrumentDetails {
    required USVString primaryDetails;
    USVString secondaryDetails;  // maybe required?
  }
```

Reply:
You would then write this like:

```
const instrument = {
  displayName: {
    primaryDetails: "FancyBank Platinum Card",
    secondaryDetails: "**** 1234 | 01/27",
  },
  icon: ...
};
```

Reply:
Ack, let's stick with the current setup then :)